libbeat/processors/add_docker_metadata/_meta/fields.yml

```diff
@@ -9,18 +9,21 @@
       type: group
       fields:
         - name: container.id
-          type: keyword
-          description: >
-            Unique container id.
+          type: alias
+          path: container.id
+          migration: true
+
         - name: container.image
-          type: keyword
-          description: >
-            Name of the image the container was built on.
+          type: alias
+          path: container.image.name
+          migration: true
```

Comment:
@webmat I introduced this not option which I think we should add to all alias fields we have created so far. This will make it possible to disable only migration alias and not necessarily disable all aliases.

Reply:
Add a checkmark in #8655 for this? I think it makes sense, yeah.

We'll need to revisit a lot of the module transitions after beta 2, so we can make this part of the tasks

Reply:
As these fields are also in https://github.com/elastic/beats/pull/9283 for 6.x we need to nail them now :-) As far as I can see this one is correct?

Reply:
Not familiar with how this field is populated. Does `docker.container.image` contain the hash-like ID, or the human readable name given to it?

Note that this also makes me notice that ECS doesn't define the field `container.image.id`.

Reply:
I haven't seen it here. It simply occurred to me that `container.image.id` is a pretty important field, when dealing with container images.

But if the old field was meant to capture the image name, then this mapping is good.